vidaud/seta
# [seta-react] Switch to Vite & add Mantine

## Description

This PR introduces the following changes:

- Switched from `react-scripts` to `Vite` for a large boost in build performance, development sever speed, and configuration capabilities
- Modified all the routes of the React application to start from `/` and changed the `nginx` configuration to support this
- Removed the `HOME_ROUTE` configuration in `seta-ui` as it's no longer required
- Now trying to access any page that doesn't exist displays the `404` message 
- Improved the `Dockerfile-dev` configuration
- Introduced and configured `Mantine` and `emotion` as the new UI and CSS-in-JS libraries
- Updated the `Footer` styling to use the new approach

## How to test

- Launch the app a navigate to `http://localhost/`
- The app should load
- You should be able to log in and logout
- The footer should look the same as before
- Accessing a page that doesn't exist (starting from the root `/`) should show the existing 404 page
